tools/topology/topology2/include/pipelines/cavs/smart-amp-playback.conf

```diff
@@ -0,0 +1,111 @@
+#
+# cAVS smart_amp playback pipeline
+#
+# A smart_amp playback pipeline. All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.smart-amp-playback.N.attribute_name"
+#
+# Usage: smart-amp-playback pipeline object can be instantiated as:
+#
+# Object.Pipeline.smart-amp-playback."N" {
+# 	format		"s16le"
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/components/copier.conf>
+<include/components/pipeline.conf>
+<include/common/audio_format.conf>
+<include/components/smart_amp.conf>
+<include/controls/bytes.conf>
+
+Class.Pipeline."smart-amp-playback" {
+
+	DefineAttribute."index" {}
+
+	<include/pipelines/pipeline-common.conf>
+
+	attributes {
+		!constructor [
+			"index"
+		]
+
+		!immutable [
+			"direction"
+		]
+
+		#
+		# smart-amp-playback objects instantiated within the same alsaconf node must have
+		# unique pipeline_id attribute
+		#
+		unique	"instance"
+	}
+
+	Object.Widget {
+		copier."1" {
+			copier_type	"host"
+			type	"aif_in"
+			num_audio_formats 3
+			# 16-bit input format 48KHz 2ch. Output sample format is always 32-bit for playback
+			Object.Base.audio_format.1 {
+				out_bit_depth		32
+				out_valid_bit_depth	32
+				dma_buffer_size "$[$ibs * 2]"
+			}
+			# 24-bit 48KHz 2ch
+			Object.Base.audio_format.2 {
+				in_bit_depth		32
+				in_valid_bit_depth	24
+				out_bit_depth		32
+				out_valid_bit_depth	32
+				dma_buffer_size "$[$obs * 2]"
+			}
+			# 32-bit 48KHz 2ch
+			Object.Base.audio_format.3 {
+				in_bit_depth		32
+				in_valid_bit_depth	32
+				out_bit_depth		32
+				out_valid_bit_depth	32
+				dma_buffer_size "$[$ibs * 2]"
+			}
+		}
+
+		pipeline."1" {
+			priority	0
+			lp_mode		0
+		}
+
+		smart_amp."1" {
```

Comment:
no this is wrong. We should have a different pipeline for host and dai, and the smart-amp should be part of the dai pipeline. Keep in mind we do want to have multiple PCMs for playback, all mixed with the result processed by the smart amp.